src/pages/about.astro

```diff
@@ -19,47 +19,84 @@ import FooterFull from '../components/footer/FooterFull.svelte'
-        content="We provide software development, data analytics, and artificial intelligence operations capabilities to MIG commanders to help them close joint, coalition, and naval kill webs. These capabilities enhance Marine commanders' decision advantage across the full spectrum of military operations."
-        image="images/marines_marathon.jpeg"
-        imageClasses="md:h-[550px] md:w-[550px] md:min-w-[550px]"
-      />
+    <div class="hidden lg:block">
+      <div class="relative z-20 pt-6">
+        <ContentImageFull
+          link="#"
+          linkText="LEARN MORE"
+          title="OUR VISION"
+          content="Upon arrival at the Marine Corps Software Factory, Marines are trained to fill the roles of a product team: product manager, software developer, and UI/UX designer. Over the course of their three-year assignment at MCSWF in Austin, Texas, they develop applications to support warfighters at the tactical edge. After their three-year tour in Austin, the teams are reassigned together to continue supporting MIG commanders throughout the fleet."
+          image="images/marine_with_ar_lens.jpeg"
+          imageClasses="md:h-[550px] md:w-[550px] md:min-w-[550px] scale-x-[-1]"
+        />
+      </div>
+      <div class="relative z-20">
+        <ContentImageFull
+          flipped
+          link="#"
+          linkText="LEARN MORE"
+          title="OUR MISSION"
+          content="We provide software development, data analytics, and artificial intelligence operations capabilities to MIG commanders to help them close joint, coalition, and naval kill webs. These capabilities enhance Marine commanders' decision advantage across the full spectrum of military operations."
+          image="images/marines_marathon.jpeg"
+          imageClasses="md:h-[550px] md:w-[550px] md:min-w-[550px]"
+        />
+      </div>
+      <div class="relative z-20">
+        <ContentImageFull
+          link="#"
+          linkText="LEARN MORE"
+          title="OUR IMPACT"
+          content="By providing Marines with the skills to develop software and implement artificial intelligence, we are giving commanders the ability to quickly produce software and AI solutions at the tactical edge."
+          image="images/marine_whiteboard.jpeg"
+          imageClasses="md:h-[550px] md:w-[550px] md:min-w-[550px]"
+        />
+      </div>
     </div>
-    <div class="relative z-20">
-      <ContentImageFull
-        link="#"
-        linkText="LEARN MORE"
-        title="OUR IMPACT"
-        content="By providing Marines with the skills to develop software and implement artificial intelligence, we are giving commanders the ability to quickly produce software and AI solutions at the tactical edge."
-        image="images/marine_whiteboard.jpeg"
-        imageClasses="md:h-[550px] md:w-[550px] md:min-w-[550px]"
-      />
-    </div>
-    <div class="relative z-20 pt-20">
-      <FooterFull client:load />
+    <div class="lg:hidden">
```

Comment:
This breakpoint should probably be md to match when the ContentImagFull swaps to mobile view.

Reply:
good catch, fixed now